exporter/otlphttpexporter/otlp.go

```diff
@@ -92,25 +99,25 @@ func (e *exporter) start(_ context.Context, host component.Host) error {
 	return nil
 }
 
-func (e *exporter) pushTraces(ctx context.Context, traces pdata.Traces) error {
-	request, err := traces.ToOtlpProtoBytes()
+func (e *exporter) pushTraces(ctx context.Context, td pdata.Traces) error {
+	request, err := tracesMarshaler.Marshal(td)
```

Comment:
Guess the plan is to move the `ToOtlpProtoBytes()` methods from the related signals?

Reply:
Hah, never mind, just saw #3434 , my bad.